apps/dashboard/src/app/(dashboard)/punishment/[id]/page.tsx

```diff
@@ -8,13 +8,13 @@ import type { FC } from "react"
 import { GenericTable } from "../../../../components/GenericTable"
 import { useEditMarkMutation } from "../../../../modules/punishment/mutations/use-edit-mark-mutation"
 import { usePersonalMarkGetByMarkId } from "../../../../modules/punishment/queries/use-personal-mark-get-by-mark-id"
-import { useUserGetQuery } from "../../../../modules/user/queries/use-user-get-query"
+import { useUserQuery } from "../../../../modules/user/queries"
 import { useMarkWriteForm } from "../write-form"
 import { useMarkDetailsContext } from "./provider"
 
 const UserNameCell: FC<{ userId: string }> = ({ userId }) => {
-  const { user } = useUserGetQuery(userId)
-  return <span>{user?.id || "Loading..."}</span>
+  const { data: user } = useUserQuery(userId)
+  return <span>{user?.auth0Id || "Loading..."}</span>
```

Comment:
E-posten er allerede unik, jeg ville egt. ikke vist auth0-id-en til noen utenfor dotkom

Reply:
Ah, I did not touch this part of the code in this PR. Looks like an N+1 query, yep.

Reply:
For the auth0 id, this does not seem like a big concern if it is shown in a dashboard for admin users? I saw some discussion about it here https://community.auth0.com/t/are-auth0-ids-senstive/71117/6. 

But I agree that the general consensus should be that you don't expose it though. This is just left over code from when we did not have other data to show maybe

Reply:
Jeg ville heller bare holdt det skjult siden det er en implementasjonsdetalj de ikke trenger vite om, og det er nok mer intuitivt å se e-posten (men atm kan jo den endres da forsåvidt). Hva de ikke vet har de ikke vondt av